frontend/src/components/Landing.vue

```diff
@@ -22,15 +22,17 @@ import { IconSize } from '@/enum/IconEnum';
                 <Button
                     class="landing-button p-button-primary"
                     label="Login with IDIR"
-                    @click="authService.methods.login"
+                    data-testid="idir-button"
```

Comment:
I feel there is no need to call it "data-testid", we can just pass an id. Shall we call them "login-idir-button" and "login-bceid-button"?

Reply:
Sure, it works as well. I'll update as suggested. Thanks.